tracing-core/src/metadata.rs

```diff
@@ -365,10 +365,13 @@ impl From<Level> for LevelFilter {
     }
 }
 
-impl Into<Option<Level>> for LevelFilter {
```

Comment:
It looks like the `Into` impl was removed? That's also a breaking change. Can we add the `From<Option<Level>> for LevelFilter` impl _without_ removing the `Into<Option<Level>> for LevelFilter` impl?

Reply:
Ah, you're right about that. My eyes skipped over this; I thought it was an `Into<LevelFilter> for Option<Level>` implementation, which meant that the reflexivity of From/Into held. Will reintroduce!